store/sqlstore/upgrade.go

```diff
@@ -865,6 +867,14 @@ func upgradeDatabaseToVersion5281(sqlStore SqlStore) {
 	}
 }
 
+func upgradeDatabaseToVersion5282(sqlStore SqlStore) {
```

Comment:
This should be in version 5.30. Not 5.28.2.

Reply:
Yes, this will be updated to reflect the appropriate version needed at the time of merge